src/odemis/driver/test/xt_client_test.py

```diff
@@ -594,6 +608,57 @@ def test_brightness_contrast(self):
         self.detector.contrast.value = init_contrast
 
 
+class TestHelperMicroscope(TestMicroscope):
+    """
+    Test the SEM connection when it's used as an extra settings control for the analog scan control
+    => No detector.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        if TEST_NOHW is True:
+            raise unittest.SkipTest("No simulator available.")
+
+        cls.microscope = xt_client.SEM(**CONFIG_SEM_HELPER)
+
+        for child in cls.microscope.children.value:
+            if child.name == CONFIG_SCANNER["name"]:
+                cls.scanner = child
+            elif child.name == CONFIG_FOCUS["name"]:
+                cls.efocus = child
+            elif child.name == CONFIG_STAGE["name"]:
+                cls.stage = child
+            elif child.name == CONFIG_CHAMBER["name"]:
+                cls.chamber = child
+
+    # Disable tests for anything related to the detector
+    def test_acquire(self):
+        pass
```

Comment:
I would personally prefer to explicitly skip these tests. If you just use `pass` it will show up as a successful test in the logs.

```suggestion
        self.skipTest("skip detector related test")
```